### PR TITLE
add counter section, improve UI overall

### PR DESCRIPTION
**Pull request by Alexandru-Bulai**
# PR  08-02-24
## Overview
In this PR we improve the UI menu and background cover... we also add the the imgs counter element to the page 
## Key Changes :
 1) gallery.html :
    - Made the background img cover the all page so the page doesn't seems to be cut off on the top 
    - Added the img counter that will return how many imgs are shown in the gallery

Screenshot 2024-02-08 at 11 52 39

### CODE REVIEW

File: public/gallery.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
 <body class="bg-blue-50">
 <script src="https://cdn.tailwindcss.com"></script>
 <link rel="stylesheet" href="CSS/gallery.css">
-
+<div id="background-img">
```
Comment (vladiiancu) [inline public/gallery.html]: Again, the name is tied too much to what it is doing. 
Secondly, there are high changes there will be other background images on a website. So the name and the fac that the id is supposed to be unique don't sit right

File: public/gallery.html [hunk 2]
```diff
@@ -89,10 +89,22 @@
...
+	<div class="bg-blue-500 text-white py-2 px-4 rounded-full shadow-lg flex items-center justify-center text-sm md:text-base lg:text-lg">
+		<svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 md:h-6 md:w-6 lg:h-8 lg:w-8 mr-2" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+			<path stroke-linecap="round"
+						stroke-linejoin="round"
+						stroke-width="1.5 md:stroke-width-2 lg:stroke-width-2"
+						d="M15 17h5l-1.5-1.5m0 0l1.5-1.5m-1.5 1.5L15 13m1.5 1.5H15m6 0a9 9 0 11-18 0 9 9 0 0118 0z" />
+		</svg>
+		<span class="font-semibold">Wildlife Tally:</span>
+		<span class="ml-2"
+					id="image-counter">123</span>
```
Comment (vladiiancu) [inline public/gallery.html:103]: What's does 123 represent?

**Review by vladiiancu — CHANGES_REQUESTED** [review]

File: public/gallery.html (hunk 2 above)
Comment (Alexandru-Bulai) [reply public/gallery.html:103]: just a placeholder to have visual effect until i implement logic with js

File: public/gallery.html (hunk 1 above)
Comment (Alexandru-Bulai) [reply public/gallery.html]: hero-img ? 
A hero section is suppose to be unique, that would be more appropiate i believe?

**Review by vladiiancu — APPROVED** [review]

**Pull request merged**